util/fibers/epoll_proactor.cc

```diff
@@ -461,10 +461,13 @@ void EpollProactor::PeriodicCb(PeriodicItem* item) {
   }
 
   item->task();
+
```

Comment:
Good catch. I actually added a test `TEST_P(ProactorTest, Periodic)` that should have covered this functionality. I wonder why it has not trigerred any issues.

Reply:
odd - it passes in CI but running locally it blocks (without this fix) - its doing `read(1, ...)` (which is `stdout`) which maybe is undefined behaviour so different on arm64...?